### PR TITLE
Add appear timeout

### PR DESCRIPTION
**Pull request by dimensi**
Hi, I had a case when I needed the appear timeout, I saw in the code that the opportunity was almost implemented, but unfinished. I hope that I completed this feature.

### CODE REVIEW

File: src/Transition.js [hunk 1]
```diff
@@ -212,7 +212,7 @@ class Transition extends React.Component {
     if (timeout != null && typeof timeout !== 'number') {
       exit = timeout.exit
       enter = timeout.enter
-      appear = timeout.appear
+      appear = timeout.appear !== undefined ? timeout.appear : enter
```
Comment (jquense) [inline src/Transition.js]: let's add a TODO here to remove this fallback on the next major bump

Comment (dimensi) [reply src/Transition.js]: added

**Comment by jquense** [comment]
thanks!

**Comment by dimensi** [comment]
thanks for fast response.

**Pull request merged**